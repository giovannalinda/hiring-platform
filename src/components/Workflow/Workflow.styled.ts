import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  margin-top: 23rem;
  margin-bottom: 26rem;
  text-align: left;

  h1 {
    font-size: ${theme.font.sizes.title};
    padding-bottom: ${theme.space[7]};
  }

  p {
    max-width: 40rem;
    font-size: ${theme.font.sizes.paragraph};
    color: ${theme.colors.gray500};
    padding-bottom: 15rem;
  }
`

export const Content = styled.div`
  display: grid;
  grid-template-columns: repeat(2, 2fr);
  grid-gap: 20rem;

  @media screen and (max-width: ${theme.breakpoints.tablet}) {
    grid-template-columns: repeat(1, 1fr);
  }
`

export const TextGroup = styled.div``
