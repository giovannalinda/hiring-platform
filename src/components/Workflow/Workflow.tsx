import Image from 'next/image'
import { Assets } from '~/assets'

import * as S from './Workflow.styled'

export function Workflow() {
  return (
    <S.Container>
      <S.Content>
        <S.TextGroup>
          <h1>Interactive human resources workflow</h1>
          <p>
            Redefining job application process and let you meet the best
            candidates in the shortest time
          </p>

          <Image src={Assets.Workflow1} width={529} height={291} />
        </S.TextGroup>

        <S.TextGroup>
          <Image src={Assets.Workflow2} width={529} height={291} />
          <h1 style={{ paddingTop: '15rem' }}>
            Innovatively built for recruiters and applicants
          </h1>
          <p>
            We create engaging experience with our dedicated tool, post an
            application and let your candidates self-checkout their way towards
            and interview
          </p>
        </S.TextGroup>
      </S.Content>
    </S.Container>
  )
}
