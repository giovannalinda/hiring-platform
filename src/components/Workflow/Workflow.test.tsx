import { render } from '~/__helpers__/app-tests'
import { Workflow } from './Workflow'

describe('Workflow', () => {
  it('should match snapshot', () => {
    const { container } = render(<Workflow />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
