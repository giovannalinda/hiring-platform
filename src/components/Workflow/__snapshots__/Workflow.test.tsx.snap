// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Workflow should match snapshot 1`] = `
<section
  class="sc-bczRLJ jGLTlh"
>
  <div
    class="sc-gsnTZi ksbbcJ"
  >
    <div
      class="sc-dkzDqf"
    >
      <h1>
        Interactive human resources workflow
      </h1>
      <p>
        Redefining job application process and let you meet the best candidates in the shortest time
      </p>
      <img
        height="291"
        src="/Workflow1.svg"
        width="529"
      />
    </div>
    <div
      class="sc-dkzDqf"
    >
      <img
        height="291"
        src="/Workflow2.svg"
        width="529"
      />
      <h1
        style="padding-top: 15rem;"
      >
        Innovatively built for recruiters and applicants
      </h1>
      <p>
        We create engaging experience with our dedicated tool, post an application and let your candidates self-checkout their way towards and interview
      </p>
    </div>
  </div>
</section>
`;
